salt/modules/yumpkg.py

```diff
@@ -2923,10 +2923,12 @@ def mod_repo(repo, basedir=None, **kwargs):
         the URL for yum to reference
     mirrorlist
         the URL for yum to reference
+    metalink
+        the URL for yum to reference
```

Comment:
can you add `.. versionadded:: 3006` for this new argument.

Reply:
Added, I think.